app/src/main/java/org/oppia/app/story/StoryActivityPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.app.story
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.domain.topic.TEST_STORY_ID_1
+import javax.inject.Inject
+
+/** The controller for [StoryActivity]. */
+@ActivityScope
+class StoryActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.story_activity)
+    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.story_fragment_placeholder,
+      StoryFragment.newInstace(TEST_STORY_ID_1)
```

Comment:
Shouldn't the story ID come from the activity intent? I'd expect that the flow is we open StoryActivity to display a specific story.

Reply:
Done